frontend/style.less

```diff
@@ -0,0 +1,389 @@
+@color_1: #747474;
```

Comment:
i think we should name these better. normally do something like:

$darkest-gray
$dark-gray
$gray
$light-gray
$lightest-gray

etc

this isn't very helpful. i'd prefer not using variables to this personally, so i can tweak and create this list later.

(it may just be quicker to keep the old CSS for now, and i can convert to LESS on my own time)